backend/canisters/local_group_index/impl/src/lifecycle/post_upgrade.rs

```diff
@@ -34,6 +40,8 @@ fn post_upgrade(args: Args) {
         utils::cycles::check_cycles_balance(user_index_canister_id)
     });
 
+    cycles_dispenser_client::init_from_bytes(&cycles_dispenser_client_state);
```

Comment:
We need to kick off the timer job in `init` too. I think that's missing from all canisters though so we can add it to all of them as a follow up PR.